river/imblearn/chebyshev.py

```diff
@@ -79,21 +87,27 @@ def predict_one(self, x):
         return self.regressor.predict_one(x)
 
     def learn_one(self, x, y, **kwargs):
-        var = self._var.get()
-        sd = var**0.5
+        self._var.update(y)
+        sd = self._var.get() ** 0.5
```

Comment:
Good catch, thanks! Let me check that right now

Reply:
@smastelini If you use Vscode, you have to check the version of Python linked to it on the bottom right when viewing a Python script. Maybe it's not the environment where you installed the last version of Black.

<img width="459" alt="image" src="https://user-images.githubusercontent.com/24591024/160411971-0183c0cc-6e8b-46d2-81e4-11976f71da72.png">

Maybe it will solve the issue 🤔

Reply:
Hmm, black seems to do some choices here.

If I get the result from var in an auxiliary variable I get:

```python
var = self._var.update(y)
sd = var**0.5
```

Adding spaces around `**` will make the hook fail. But, if I get the the result directly, then the formatting becomes `sd = self._var.get() ** 0.5`. Removing the space will make the hook also fail.

Interesting choices. Look at the PR title in black's repo: [Hug power operators if its operands are "simple"](https://github.com/psf/black/pull/2726).

Reply:
Thanks for the reply @raphaelsty! Actually, I ran everything on my terminal hahah

I got used to doing everything this way, git, pre-commit hooks and so on :)